backbone-despotism.js

```diff
@@ -42,6 +45,15 @@
   }
 } (this, function(exports, Backbone, _) {
   Backbone.StrictModel = Backbone.Model.extend({
+    constructor: function(attributes, options) {
+      // Make sure that the options object when initializing a new instance of
+      // the StrictModel always has { initialize: true }
+      attributes = attributes || (attributes = {});
+      options || (options = {});
```

Comment:
And isn't the same style here just a liiiiittle bit clearer? 😜  (I like when it's obvious when you're declaring something.)